build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext.androidBuildTool = '29.0.2'
-    ext.androidx_appcompat = '1.1.0'
+    ext.androidx_appcompat = '1.2.0-rc01'
```

Comment:
Unless blocking, let's try to use only the stable versions.

Reply:
The stable version `1.1.0` has a [bug](https://stackoverflow.com/questions/58028821/webview-crash-on-android-5-5-1-api-21-22-resourcesnotfoundexception-string-r) causing crash on android 5 when passing activity context.  We have only option to update the library.

Reply:
I read your previous comment just now. Thanks @minh-rakuten

Reply:
Updated on USERGUIDE.md